test/goss/remote/vars/seed-peers-unstable.yaml

```diff
@@ -1,3 +0,0 @@
-hosts:
```

Comment:
Removal of ansible/group_vars/tag_env_unstable/vars.yml ?

Reply:
Thanks for noticing, removed in 41c7f9e